docs/essentials/backgrounds.mdx

```diff
@@ -197,8 +246,28 @@ Default: `0.5`
 
 The opacity of the grid lines.
 
+<Callout variant="warning" icon="🆕">
+
+  **With the [`globals` API](#globals-api)**
+
+  #### `options`
+
+  Type: `Record<string, { name: string; value: string }>`
+
+  Required: See description
+
+  Defines the background colors. Replaces [`values`](#values). See above for a [usage example](#configuration).
+
+  Required when defining `backgrounds` parameter at project level.
+
+</Callout>
+
 #### `values`
 
 Type: `{ name: string; value: string }[]`
 
-Available background colors. See above for a [usage example](#configuration). Required if you haven't opted into the [globals API](#overriding-backgrounds-with-globals).
+Required: See description
+
+Available background colors. See above for a [usage example](#configuration).
+
+Required when defining `backgrounds` parameter at project level.
```

Comment:
There's a typo here, should be something like:
```suggestion
Required when defining the `backgrounds` parameter at the project level.
```
Also, this is not entirely true, as documented in the examples above. Users can define this at the component and story level.

Reply:
> Also, this is not entirely true, as documented in the examples above. Users can define this at the component and story level.

This sentence isn't about when `parameters.backgrounds.values` can be defined. It's about when it _must_ be defined, i.e. when it is required.

Would you find this more clear?

```suggestion
When defining the `backgrounds` parameter at the project level (in `.storybook/preview.js|ts`), you _must_ define the `values` property.
```

Reply:
That's more accurate.